pallets/transaction-payment/src/lib.rs

```diff
@@ -483,6 +484,23 @@ where
         T::CddHandler::set_payer_context(Some(payer_key));
         Ok((fee, Some(imbalance)))
     }
+
+    /// Tip in `DispatchClass::Normal` transactions MUST be zero.
+    fn ensure_valid_tip(
+        &self,
+        info: &DispatchInfoOf<T::Call>,
+    ) -> Result<BalanceOf<T>, TransactionValidityError> {
+        if let DispatchClass::Normal = info.class {
+            ensure!(
+                self.0 == Zero::zero(),
+                TransactionValidityError::Invalid(InvalidTransaction::Custom(
+                    TransactionError::ZeroTip as u8
+                )),
```

Comment:
Here we are not considering that only CDD and GC can tip the operational transactions.

Reply:
That is going to be addressed in [MESH-1520](https://polymath.atlassian.net/browse/MESH-1520) :+1: